backend/lucene/src/main/java/org/hibernate/search/backend/lucene/analysis/impl/ScopedAnalyzer.java

```diff
@@ -10,21 +10,22 @@
 import java.util.Map;
 
 import org.apache.lucene.analysis.Analyzer;
-import org.apache.lucene.analysis.AnalyzerWrapper;
+import org.apache.lucene.analysis.DelegatingAnalyzerWrapper;
+import org.apache.lucene.analysis.core.KeywordAnalyzer;
+
 import org.hibernate.search.util.common.impl.CollectionHelper;
 
 /**
  * @author Guillaume Smet
  */
-public final class ScopedAnalyzer extends AnalyzerWrapper {
+public final class ScopedAnalyzer extends DelegatingAnalyzerWrapper {
```

Comment:
Very good simplification here. There's always time to make things more complex ;)